tests/test_tools.py

```diff
@@ -2009,3 +2010,155 @@ def test_remove_duplicate_docs(inp, expected):
 def test_filter_employment_for_advisees(inp, expected):
     actual = filter_employment_for_advisees(inp[0], inp[1], "ms", dt.date(2021,6,3))
     assert actual == expected
+
+person1 = {"_id":"tstark", "aka":"iron man", "name":"tony stark"}
+person2 = {"_id":"nromanov", "aka":"black widow", "name":"natasha romanov"}
+PEOPLE = [person1, person2]
+
+presentation1 = {"_id":"abc", "authors":["tstark"], "begin_year":2018, "begin_month":1, "begin_day":1,
+                 "department":"apam", "institution":"columbiau", "status":"accepted", "type":"award"}
+presentation2 = {"_id":"ghi", "authors":["tstark","nromanov"], "begin_year":2019, "begin_month":1, "begin_day":2,
+                 "end_year":2019, "end_month":1, "end_day":8, "department":"physics", "institution":"rutgersu",
+                 "status":"cancelled", "type":"poster"}
+PRESENTATIONS = [presentation1, presentation2]
+
+institution1 = {"_id":"columbiau", "city":"New York", "country":"USA", "name":"Columbia University", "state":"NY"}
+institution2 = {"_id":"rutgersu", "city":"New Brunswick", "country":"USA", "name":"Rutgers University", "state":"NJ"}
+INSTITUTIONS = [institution1, institution2]
+
+@pytest.mark.parametrize(
```

Comment:
please put this line right above the test with `@pytest.mark.parametrize("args, kwargs, expected", pres_map)` (see above other tests for the pattern)